### DISCUSSION TITLE
Renovate container with bitbucket

### DISCUSSION THREAD

**Discussion by lernwithshubh** [renovatebot/renovate#21995] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

I want to work with renovate container with bitbucket, but cannot find official documentation much helpful, Can anyone assist with easy documentation to implement 

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
You can search for "bitbucket" on docs.renovatebot.com

**Reply by lernwithshubh** [reply]
that is not making things clear, can you assist with simple implementation?

**Reply by rarkins** [reply]
https://docs.renovatebot.com/getting-started/running/

**Reply by lernwithshubh** [reply]
I want to use renovate docker container to work with repos on bitbucket, not getting that how to get it done.

**Reply by rarkins** [reply]
What have you tried so far?

**Reply by lernwithshubh** [reply]
I have created the container on my local with renovate image but dont know how to proceed and use it

**Reply by rarkins** [reply]
It looks like you are hoping for someone to guide you every step of the way but unfortunately I cannot dedicate the time required for that

**Reply by lernwithshubh** [reply]
Any documentation on how to use renovate container with bitbucket?

**Reply by viceice** [reply]
https://github.com/renovatebot/docker-renovate/blob/main/docs/bitbucket.md#samples-for-bitbucket-pipelines
